src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -198,8 +211,9 @@ def set_opts(self, curl, url, params, headers,
         if cert:
             curl.setopt(pycurl.SSLCERT, cert)
         if verbose:
-            curl.setopt(pycurl.VERBOSE, 1)
+            curl.setopt(pycurl.VERBOSE, True)
             curl.setopt(pycurl.DEBUGFUNCTION, self.debug)
+        curl.setopt(pycurl.VERBOSE, True)
```

Comment:
I'm not sure you want VERBOSE level all the time, that why it was under `if verbose` block.

Reply:
Haaa, I was looking for where the verbosity was coming from. Thanks, it looks like this line sneaked in.